gulpfile.js

```diff
@@ -60,25 +55,144 @@ gulp.task('collectTeamInfo', function(callbackFinish){
 			members.push(member);
 
 			//resize the image to a 480x480
-			Jimp.read(remoteFileName).then(function (imageObject) {
-				imageObject.cover(480,480).write("./static/assets/images/team-images/"+filename);
+			Jimp.read(remoteFileName.trim()).then(function (imageObject) {
+				imageObject.cover(480,480).write(team_image_folder+filename);
 				//increment the progressbar
 				progress_bar.increment();
 				finishedTasks++;				
 				if (totalTasks == finishedTasks) {
-					writeYaml(members);
-					progress_bar.stop();
+					console.log("\nDone! Storing team YAML");
+					yamlString = YAML.stringify(members, 4);
+					fs.writeFile("./data/Team.yaml", yamlString, function(err) {
+						progress_bar.stop();						
+						callbackFinish();
+					});
 				}
 			}).catch(function (err) {
 			});
 		});
 	});
 });
 
+//generate an image
+gulp.task('generateTeamOpengraph', ['collectTeamInfo'], function(callbackFinish){ 
```

Comment:
i feel like this function is large enough to move it out into a separate file somewhere?

Reply:
Is moving JS outside the Gulpfile a thing? I can't seem to find an accepted way to do so.

We could also leave it in there. The gulpfile isn't going to get bigger

Reply:
alright yeah, leave it in ;)

Reply:
Agree'd! It's now done.